services/QuillLMS/config/initializers/rails_admin.rb

```diff
@@ -2,6 +2,7 @@
 
 Rails.application.config.eager_load do
   RailsAdmin.config do |config|
+    config.asset_source = :webpacker
```

Comment:
I [upgraded](https://github.com/empirical-org/Empirical-Core/pull/10812) RailsAdmin to 3.1.2 in preparation for the Rails 7 upgrade a little over a month ago.  AFAICT, the warning about asset source is incorrect and it still exists on this branch even after adding the asset_source = :webpacker.  (e.g. run rails console in your local environment):

![Screenshot 2023-09-18 at 8 16 00 AM](https://github.com/empirical-org/Empirical-Core/assets/2057805/19d35e14-d82f-4bfc-a0e0-27b05741ef16)

Unless there is something in the Rails Admin that you noticed not working properly, I would recommend removing the following changes in this PR:
```
services/QuillLMS/app/javascript/packs/rails_admin.js
services/QuillLMS/app/javascript/stylesheets/rails_admin.scss
services/QuillLMS/config/initializers/rails_admin.rb
services/QuillLMS/package.json
```
since I don't think they do anything.